### PR TITLE
Rename and create admin-tools page and routing from scans

### LINKED ISSUE #133: Update Scan Page and Address Dependencies
# 💡 Summary #

The current state of the project includes the “Web Page Scan” feature, which is considered out of scope for our project goals. Additionally, we need to ensure that all dependencies, especially related to routing, are properly addressed.

## Motivation and context ##

Eliminates confusion to the process to pull vulnerability data for organizations.

## Implementation notes ##

- Change Scans page to Admin Tools (Only “Global Admin” Accessible)
- Update header by moving link to right-left dropdown menu
- Add Administration tools for site maintenance to Admins Tools page
- Remove any other code dependencies

## Acceptance criteria ##

How do we know when this work is done?

- [ ] Add simple screen shot to ticket for proof
- [ ] Ensure that all necessary dependencies are addressed
- [ ] Update Playwright Tests

### PR DESCRIPTION
**Pull request by ameliav**
#  #

## 🗣 Description ##
Rename and create admin-tools page and routing from scans.

## 💭 Motivation and context ##
The current state of the project includes the “Web Page Scan” feature, which is considered out of scope for our project goals. Additionally, we need to ensure that all dependencies, especially related to routing, are properly addressed.

## 🧪 Testing ##

- Change Scans page to Admin Tools (Only “Global Admin” Accessible)
- Update header by moving link to right-left dropdown menu
- No longer any routes directly to /scans
- /scans -> /admin-tools

## 📷 Screenshots ##

Scans is now under "Admin Tools"
image

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [ ] Revert dependencies to default branches.
- [ ] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Create a release.

### CODE REVIEW

**Review by nickviola — APPROVED** [review]
LGTM

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -167,21 +168,26 @@ const App: React.FC = () => (
                     permissions={['standard', 'globalView']}
                   />
                   <RouteGuard
-                    path="/scans"
+                    path="/admin-tools/scans"
+                    component={AdminTools}
+                    // permissions={['standard', 'globalView']}
```
Comment (cduhn17) [inline frontend/src/App.tsx]: Remove commented out code if not needed, and if needed place a TODO and/or make issue for what the commented out code might be used for in the future.

**Review by cduhn17 — COMMENTED** [review]
See comments. There are multiple locations where there is commented out code.

File: frontend/src/App.tsx (hunk 1 above)
Comment (ameliav) [reply frontend/src/App.tsx]: No problem, removed it in a new commit: df615b5cf6b074329def392f28473d113024ca21

**Review by schmelz21 — APPROVED** [review]
This now looks good.

**Pull request merged**